pkg/jobrunaggregator/jobtableprimer/migration_test.go

```diff
@@ -1,238 +0,0 @@
-package jobtableprimer
-
-import (
-	"reflect"
-	"testing"
-
-	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
-)
-
-func TestMigrationOfJobAttributes(t *testing.T) {
```

Comment:
I only created this to be sure that I got the same content I had manually entered.  it's no longer needed since we create via command now.